sdk/eventhub/test-resources.json

```diff
@@ -57,7 +57,7 @@
     },
     "perTestExecutionLimitMinutes": {
       "type": "string",
-      "defaultValue": "10",
+      "defaultValue": "15",
```

Comment:
Only necessary for the China cloud, but there is no current way to infer the cloud beyond sniffing the data center and manually translating.   This shouldn't be an issue, but if we have concerns, I can look into passing the cloud through to the test resources template.